.github/workflows/release.yaml

```diff
@@ -17,7 +17,10 @@ jobs:
     steps:
     - name: Checkout
       uses: actions/checkout@v3
-
+    - name: Get history and tags for SCM versioning to work
+      run: |
+        git fetch --prune --unshallow
+        git fetch --depth=1 origin +refs/tags/*:refs/tags/*
```

Comment:
No need for this step. The `actions/checkout` has an attribute that will pull this. Like so:
```
- name: Checkout repo
  uses: actions/checkout@v3
  with:
    fetch-depth: 0 # Fetch all history for all branches and tags.
```